packages/apps/human-app/frontend/src/pages/worker/jobs/components/available-jobs/desktop/available-jobs-table.tsx

```diff
@@ -139,10 +141,16 @@ const getColumns = ({
           <Grid sx={{ display: 'flex', justifyContent: 'flex-end' }}>
             <Grid sx={{ width: '5rem', height: '2.5rem', padding: '0.2rem 0' }}>
               <TableButton
-                loading={loadingButtons}
+                loading={
+                  pendingAssignmentEscrowAddress === escrow_address &&
+                  assignmentPending
+                }
                 onClick={() => {
                   assignJob({ escrow_address, chain_id });
                 }}
+                sx={{
+                  width: '94px',
```

Comment:
Is this intentional?

Reply:
yep, I just want to have fixed with for both states to prevent resizing